src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,138 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    private String genre;
+
+    @Column(nullable = false)
```

Comment:
We recommend using `javax.validation.constraints.NotNull` because it checks that the value is not null **before** getting to the persistance layer 🙂

Reply:
Ohhh I see, i thought the same but in the steps given they did not tell to install validation dependency and for that reason i did not use it, and i put something that could be include without installing anything else